doc/Command-Reference.md

```diff
@@ -712,7 +712,25 @@ This command displays the cause of the previous reboot
   admin@sonic:~$ show reboot-cause
   User issued reboot command [User: admin, Time: Mon Mar 25 01:02:03 UTC 2019]
   ```
+#### Applicable only to SmartSwitch platforms
+**show reboot-cause all**
 
+This command displays the cause of the previous reboot for the Switch and the enabled DPUs
+
+- Usage:
+  ```
+  show reboot-cause all
+  ```
+
+- Example:
+  ```
+  root@MtFuji:~$  show reboot-cause all
```

Comment:
@rameshraghupathy This will list the DPUs that are in admin shut?

Reply:
No.  This will show the "show reboot-cause" output of the DPUs that are admin UP.

Reply:
@prgeor

Reply:
@rameshraghupathy is this still true? With new design change, NPU can still find the last reboot cause of the DPU irrespective of its ADMIN state.

Reply:
@prgeor  With the new design NPU can still find the last reboot cause of the DPU irrespective of its ADMIN state.
@vvolam Done